ci/build_cpp.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
 
 set -euo pipefail
 
 source rapids-env-update
 
 export CMAKE_GENERATOR=Ninja
+BUILD_PATH=/opt/conda/conda-bld/*libraft-split*/cpp/build/
```

Comment:
```suggestion
BUILD_PATH=$(echo /opt/conda/conda-bld/*libraft-split*/cpp/build/)
```

I learned earlier in this PR that globs aren't expanded via bash when they're simply assigned to a variable.

Wrapping it like this seemed to resolve that. See my previous commit below for an example.


- https://github.com/rapidsai/raft/pull/1240/commits/3b2ceac292ea3157fe65fc14e31469690a5806be